academy/4-my-own-chain/wager-denom.md

```diff
@@ -101,27 +101,31 @@ You have prepared the ground by placing this token denomination into the relevan
 
 ## Live testing with a relayer
 
-With the checkers application ready to accommodate IBC-foreign tokens, you want to run some tests locally with another blockchain's tokens, without running a large scale operation. Luckily, Starport has built in the [Typescript relayer](https://docs.starport.com/kb/relayer.html). If you look at the GUI Starport created in your checkers blockchain, you will see a _Relayers_ section on the left.
+With the checkers application ready to accommodate IBC-foreign tokens, you should run some tests locally with another blockchain's tokens without running a large-scale operation. Starport luckily has built in the [Typescript relayer](https://docs.starport.com/kb/relayer.html). If you look at the GUI Starport created in your checkers blockchain, you will see a _Relayers_ section on the left.
 
-As you recall, a relayer is a process that transfers IBC packets between two blockchains. It just so happens that here this process is **running in your browser**, using the account you configured in your browser. This account in the browser is the same one you would use to play a game of checkers. Let's dub it `alice123@checkers`.
+A relayer is a process that transfers IBC packets between two blockchains. It just so happens that here this process is **running in your browser** using the account you configured in your browser. The account is the same one you would use to play a game of checkers. Name it `alice123@checkers`.
 
-1. On the checkers end, the relayer is already configured to connect to your running checkers blockchain, and to use the tokens of whichever account you have configured in your browser, here `alice123@checkers`. In this sense, it gets the same privileges to access your tokens that you have granted to the checkers browser application.
-2. You now need to configure it to connect to the other blockchain, the one that hosts the foreign tokens you want to transfer. It can be the Cosmos Hub, or a [testnet](https://tutorials.cosmos.network/connecting-to-testnet/testnet-tutorial.html) that you or someone else runs.
-3. You also need to fund the relayer's account on the remote chain so that it can operate at all. This account is generated from the same private key as `alice123@checkers`, so let's call it `alice465@remote`. The relayer shows you in the browser which account this is.
+1. On the checkers end, the relayer is already configured to connect to your running checkers blockchain and to use the tokens of whichever account you have configured in your browser, here `alice123@checkers`. It gets the same privileges to access your tokens that you have granted to the checkers' browser application.
```

Comment:
I think a "Therefore it gets the same..." would assist comprehension. Because the link between these 2 sentences is truly a therefore. So either we write it so that it snaps in the reader's mind as they read, or we keep it vague and wait for the reader to have read these 2 sentences, pause 5 seconds and think "oh wait one follows from the other".